helm/alfresco-content-services/community_values.yaml

```diff
@@ -71,6 +71,10 @@ repository:
       memory: "3000Mi"
     limits:
       memory: "3000Mi"
+  # -- The startup probe to cover the worse case startup time for slow start containers
+  startupProbe:
+    periodSeconds: 30
+    failureThreshold: 20
```

Comment:
I would reduce to 10 and have a 5m wait for the slow container startup at most, WDYT